src/Fantomas.Tests/CommentTests.fs

```diff
@@ -936,6 +936,27 @@ type substring =
 #endif
 """
 
+[<Test>]
+let ``Single block comment in namespace, 1951`` () =
```

Comment:
unit tests should start with [a lowercase letter.](https://github.com/fsprojects/fantomas/blob/master/CONTRIBUTING.md#guidelines)